### DISCUSSION TITLE
Kudos to the Freeplane team.

### DISCUSSION THREAD

**Discussion by sheepdoggie** [freeplane/freeplane#543] [User testimonials]
I was blind-sided but many of the changes when going from Freeplane 1.8.11 to 1.10.3, because the changes are major. But now that I have been working with it I have nothing but praise for the changes and additions. I have been using this software every day for the last 5 years and would not be able to function without it. But the latest version is amazingly good. So useful and helpful and intelligently designed. Bravo!

**Comment by Newman5** [comment]
Just wanna underline what Sheepdoggie says.  I am so glad Freemind and the .mm format is alive and well.

**Comment by koellikerr** [comment]
Just came across Feeplane recently. Absolutely love it. Thanks and keep up the great work!

**Comment by LukeMW** [comment]
I also want to jump on the praise bandwagon.  I discovered Freeplane some time ago and I use it every day, for a multitude of things, I love it.  Thanks to the devs for the great work.
